specification/network/resource-manager/readme.md

```diff
@@ -2232,6 +2232,7 @@ swagger-to-sdk:
     after_scripts:
       - bundle install && rake arm:regen_all_profiles['azure_mgmt_network']
   - repo: azure-resource-manager-schemas
+  - repo: azure-cli-extensions
```

Comment:
Confirm with the CLI extension owner and CLI codegen is disabled in this PR https://github.com/Azure/azure-rest-api-specs/pull/15862/files, no need to configure this anymore